docs/guidelines/components/tabs.md

```diff
@@ -0,0 +1,35 @@
+# Tabs
+
+We use Tabs component to offer structured views of information to users.
+
+###<div class="mgt-1">Standard tabs</div>
+<div class="mgt-2 ez-guidelines-tabs">
+[[code_example {html}
+<ul class="nav nav-tabs ez-tabs" role="tablist" id="ez-tab-list-location-view">
+    <li class="nav-item">
+        <a class="nav-link active" id="ez-tab-label-location-view" data-toggle="tab" href="#ez-tab-location-view-content" role="tab" aria-controls="ez-tab-location-view-content" aria-expanded="1">View</a>
```

Comment:
The `id` can be dropped from here as well as `href` attribute. In general, the tab comes from Bootstrap 4. There's nothing eZ-specific in the HTML code structure.
Please update the HTML code by removing not required parts :)